src/bin/hb-service.ts

```diff
@@ -803,7 +807,7 @@ export class HomebridgeServiceHelper {
    * Creates the default config.json
    */
   public async createDefaultConfig() {
-    await fs.writeJson(process.env.UIX_CONFIG_PATH, {
+    await fs.outputJson(process.env.UIX_CONFIG_PATH, {
```

Comment:
Any reason for this change?

The storagePathCheck is always called first so the path should exist?

Reply:
UIX_CONFIG_PATH and the storage path seem to not always be the same. This was for symmetry.